pyphi/exceptions.py

```diff
@@ -21,4 +21,4 @@ class JSONVersionError(ValueError):
 
 
 class WrongDirectionError(ValueError):
-    """The wrong direction was provided."""
+    """The wrong direction was provided."""
```

Comment:
Make sure your IDE is set up to have trailing newlines on all files. See comment below about the `black` formatter.